src/client/src/orangehrmPerformancePlugin/pages/EmployeeTrackerLogs.vue

```diff
@@ -0,0 +1,529 @@
+                  {{ $t('performance.added_on') + ': ' + item.addedDate }}
+                </oxd-text>
+                <oxd-text v-if="item.modifiedDate">
+                  {{ $t('performance.modified_on') + ': ' + item.modifiedDate }}
+                </oxd-text>
+              </div>
+            </div>
+          </div>
+        </oxd-sheet>
+        <oxd-loading-spinner
+          v-if="isLoading"
+          class="orangehrm-container-loader"
+        />
+      </oxd-grid>
+    </div>
+    <add-tracker-log-modal
+      v-if="showAddTrackerModal"
+      :tracker-id="trackerId"
+      @close="onAddTrackerModalClose"
+    ></add-tracker-log-modal>
+    <edit-tracker-log-modal
+      v-if="showEditTrackerModal"
+      :tracker-id="trackerId"
+      :tracker-log-id="editTrackerLogId"
+      @close="onEditTrackerModalClose"
+    ></edit-tracker-log-modal>
+  </div>
+</template>
+
+<script>
+import {APIService} from '@/core/util/services/api.service';
+import {reactive, toRefs} from 'vue';
+import useInfiniteScroll from '@/core/util/composable/useInfiniteScroll';
+import Icon from '@ohrm/oxd/core/components/Icon/Icon.vue';
+import Sheet from '@ohrm/oxd/core/components/Sheet/Sheet';
+import Spinner from '@ohrm/oxd/core/components/Loader/Spinner';
+import Dropdown from '@ohrm/oxd/core/components/CardTable/Cell/Dropdown.vue';
+import AddTrackerLogModal from '@/orangehrmPerformancePlugin/components/AddTrackerLogModal';
+import EditTrackerLogModal from '@/orangehrmPerformancePlugin/components/EditTrackerLogModal';
+
+export default {
+  name: 'ViewEmployeeTrackerLogs',
+  components: {
+    'oxd-icon': Icon,
+    'oxd-sheet': Sheet,
+    'oxd-loading-spinner': Spinner,
+    'oxd-dropdown': Dropdown,
+    'add-tracker-log-modal': AddTrackerLogModal,
+    'edit-tracker-log-modal': EditTrackerLogModal,
+  },
+  props: {
+    trackerId: {
+      type: Number,
+      required: true,
+    },
+  },
+  setup(props) {
+    // TODO change to window.appGlobal.baseUrl
+    const http = new APIService(
+      'https://942be86c-56c6-42e3-ac85-874a20c7ce9b.mock.pstmn.io',
+      `api/v2/performance/trackers/${props.trackerId}/logs`,
+    );
+    const limit = 10;
+    const state = reactive({
+      total: 0,
+      items: [],
+      meta: {
+        positive: 0,
+        negative: 0,
+      },
+      infinite: false,
+      isLoading: false,
+    });
+
+    const fetchData = () => {
+      state.isLoading = true;
+      http
+        .getAll({
+          limit: limit,
+          offset: state.items.length === 0 ? 0 : limit,
+        })
+        .then(response => {
+          const {data, meta} = response.data;
+          state.total = meta?.total || 0;
+          if (Array.isArray(data)) {
+            state.items = [
+              ...state.items,
+              ...data.map(item => {
+                return {
+                  ...item,
+                  reviewerPictureSrc: `${window.appGlobal.baseUrl}/pim/viewPhoto/empNumber/${item.reviewer.empNumber}`,
```

Comment:
`pim/viewPhoto/empNumber` @RajithaKumara we might have to exempt this route for user permissions?